Dockerfile

```diff
@@ -3,6 +3,20 @@ FROM bitwalker/alpine-elixir:1.4.2
 MAINTAINER PRX <sysadmin@prx.org>
 LABEL org.prx.app="yes"
 
+# install git, aws-cli
+RUN apk update && apk --update add \
+    ca-certificates \
+    git
+
+RUN apk --no-cache update && \
+    apk --no-cache add python py-pip py-setuptools ca-certificates groff less && \
+    pip --no-cache-dir install awscli && \
```

Comment:
Looks like there is an alpine package for aws-cli, in the edge/testing repo.  So you _could_ trim this down to

```
RUN echo http://nl.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
    apk --no-cache add aws-cli
```

And since alpine-elixir uses alpine > 3.3, you [don't actually need](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache) the `--update` or `rm -rf` stuff.